sdk/modelparser/azure-digitaltwins-parser/src/main/java/com/azure/digitaltwins/parser/ParserServiceVersion.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.digitaltwins.parser;
+
+import com.azure.core.util.ServiceVersion;
+
+/**
+ * The service API versions of Azure DigitalTwins Model Parser that are supported by this client.
+ */
+public enum ParserServiceVersion implements ServiceVersion {
+    V2021_02_11("2021_02_11");
```

Comment:
Is this a bogus API version just to meet track2 guidelines?

What value is used in the other languages?

Reply:
we are the pioneers at the moment. There is no API call at all so I think we can get away with not having any API versions at all.